classes/helper.php

```diff
@@ -805,20 +812,28 @@ public static function handle_replace_csv(string $data, string $type = 'db') {
                     'mimetype' => $record[$mimeindex],
                 ];
 
-                if (!self::replace_text_in_file($filerecord, $record[$matchindex], $record[$replaceindex],
-                    $record[$internalindex])) {
-                    $rowskip++;
-                }
+                self::replace_text_in_file($filerecord, $record[$matchindex], $record[$replaceindex],
+                    $record[$internalindex], $rowcounts);
             }
-
+            $totalrows++;
             // Update the progress bar.
-            $rowcount++;
-            $progress->update_full(100 * $rowcount / $contentcount,
-                "Replaced $rowcount rows. Skipped $rowskip rows. Error replacing $rowerror rows.");
+            $progress->update_full(
+                100 * $totalrows / $contentcount,
```

Comment:
can you refactor this long string with the same long string below?

also I think its too long, needs to be < 80 chars